src/test/lambda/explorer/errorNode.test.ts

```diff
@@ -5,8 +5,8 @@
 
 import * as assert from 'assert'
 import { DefaultRegionNode } from '../../../lambda/explorer/defaultRegionNode'
-import { ErrorNode } from '../../../lambda/explorer/errorNode'
 import { RegionInfo } from '../../../shared/regions/regionInfo'
+import { ErrorNode } from '../../../shared/treeview/nodes/errorNode'
```

Comment:
errornode.ts moved, this file (errorNode.test.ts) should move in tandem

Reply:
Please check all relocated files for test-file counterparts, to see if any require moving also

Reply:
* Moved errorNode.test.ts and removed reliance on non-test code (defaultRegionNode)
* Left regionNode in place and renamed to defaultRegionNode since regionNode is an untestable interface.